src/client/Dashboard.tsx

```diff
@@ -904,7 +1075,7 @@ class Dashboard<Props> extends React.Component<Props, IDashboardState> {
         if (queryParams !== "") {
           const promise = HttpHelper.httpGet("/api/author?" + queryParams);
 
-          promise.then(data => {
+          promise.then((data: any) => {
```

Comment:
Out of curiosity why do we need to add `any`?

Reply:
I was getting a linting error